test/let_value_test.cpp

```diff
@@ -281,3 +284,9 @@ TEST(Let, LetValueWithTraitlessPredecessor) {
   ASSERT_TRUE(ret);
   EXPECT_EQ(*ret, 42);
 }
+
+TEST(Let, LvalueConnectable) {
+  sync_wait(repeat_effect_until(
+      let_value(let_done(just(), [] { return just(); }), [] { return just(); }),
+    [n=0]() mutable noexcept { return n++ == 1000; }));
+}
```

Comment:
I think the Windows CI that failed is blowing the stack here somehow.  The SEH exception that's being reported seems to correspond to `STATUS_STACK_OVERFLOW`.

If I understand the intent of this test correctly, you don't need 1,000 iterations, just more than 1. Maybe lower the iteration count and see if the test passes?
```suggestion
TEST(Let, LvalueConnectable) {
  sync_wait(repeat_effect_until(
      let_value(let_done(just(), [] { return just(); }), [] { return just(); }),
    [n=0]() mutable noexcept { return n++ == 5; }));
}
```